### ISSUE TITLE
Check collisions between two entities

### ISSUE THREAD

**Issue by musjj** [Jondolf/bevy_xpbd#164]
How do I check for collisions between two entities? The docs has this:

```rust
fn my_system(mut collision_event_reader: EventReader<Collision>) {
    for Collision(contact) in collision_event_reader.iter() {
        println!("{:?} and {:?} are colliding", contact.entity1, contact.entity2);
    }
}
```

But how do I check between two *specific* entities? This is the ideal API I am imagining:

```rust
fn my_system(
    player_query: Query<Entity, With<Player>>,
    bullet_query: Query<Entity, With<Bullet>>,
) {
    let player = player_query.single();
    for bullet in bullet_query.iter() {
        if let Collision(contact) = check_collision(player, bullet) {
            do_something(contact);
        }
    }
}
```

Is there a way to do this with this library?

**Comment by Jondolf** [comment]
Right now, you would need to iterate through collision events and manually find the collision with the correct entities. This isn't ideal of course, but I don't think there's a better way yet.

However, this will be much nicer in 0.3. On the main branch there is a new `Collisions` resource that allows you to do `collisions.get(entity1, entity2)`, which should be what you want. This is somewhat WIP since there's still some issues, but they should be fixed by #112 once I have time to finish it.

**Comment by Jondolf** [comment]
I'll close this since `Collisions` is now properly working and usable on the main branch. It has `get` and `get_mut` which should work for your use case.
